portfolio/src/main/webapp/comments-script.js

```diff
@@ -0,0 +1,7 @@
+
+function callFetch(){
+    fetch("/data")
+    .then(res => res.json())
+    .then(res => console.log(res))
+    .catch(err => console.err(err));
```

Comment:
Add a TODO to indicate you haven't implement inserting into view yet.

Reply:
Will implement. Resolved